src/components/reservation/forms/ModalReservationFormSelect.tsx

```diff
@@ -32,7 +33,8 @@ const ModalReservationFormSelect = ({
   header,
   items,
   defaultSelectedItem,
-  selectHandler
+  selectHandler,
+  ariaLabel = "modal-reservation-form-select"
```

Comment:
Please reconsider making `ariaLabel` optional.

I do not think this makes for a suitable default value to be handled by screen readers.

Reply:
You're right, it's an absolutely terrible value. I for some reason thought I was working with our good old dataCy props.